Practice Areas/Engineering/code_review_guidelines.md

```diff
@@ -12,12 +12,21 @@ Before sending your code to a VSP team for review, your code should meet all of
 ## Guidelines
 
 * **Review lifecycle**
-    * Before sending code to VSP team, have it reviewed by your team (see **First review** below).
-    * An engineer from the VSP will aim to review within 24 hours (business days) and each additional round of changes will be reviewed within 24 hours.  If you need it faster, escalate to Leah and Patrick.
+    * Before sending code to VSP team have it reviewed by your team (see **First review** below) using a [Draft Pull Request](https://help.github.com/en/articles/about-pull-requests#draft-pull-requests).  
+      * This prevents [code owners](https://help.github.com/en/articles/about-code-owners) from being notified before pull request is ready for review.
+    * After having your pull request reviewed by your team you can mark your pull request as ready for review.
+      * An engineer from the VSP will aim to review within 24 hours (business days) and each additional round of changes will be reviewed within 24 hours.  If you need it faster, escalate to your lead or scrum master.
```

Comment:
I like the idea of going through the lead or scrum master, but I think it still ultimately needs to go through Leah or Patrick.

Does that sound right, @gunsch?

Reply:
to leave the wording more general, let's say "escalate via your DSVA product owner"

Reply:
changed